### PR TITLE
Lint remaining packages

### PR DESCRIPTION
**Pull request by remi-stripe**
This lint the remaining API packages. It also fixes a missing getter in the `topup` package

r? @brandur-stripe 
cc @stripe/api-libraries

### CODE REVIEW

File: threedsecure/client.go [hunk 1]
```diff
@@ -13,24 +13,28 @@ type Client struct {
 	Key string
 }
 
-// New POSTs new 3D Secure auths.
-// For more details see https://stripe.com/docs/api#create_three_d_secure.
+// This package is deprecated and should not be used anymore.
+// It is still here for backwards compatibility for now.
+// To use 3D Secure, please use the source package instead.
```
Comment (brandur-stripe) [inline threedsecure/client.go]: Minor, but in Go you can actually document the packages directly. Would you mind moving this documentation up on top of `package theedsecure` above? That way it'll show up in rendered Godoc, etc.

**Comment by brandur-stripe** [comment]
Wow. you got all these implemented in < 1 day! Amazing haha. Thanks for staying with it.

Left a minor comment above, but otherwise LGTM.

**Comment by remi-stripe** [comment]
@brandur-stripe thank you for the quick review! This can totally wait tomorrow by the way, but I fixed the docs for that package

**Comment by brandur-stripe** [comment]
No worries! Thanks for the fix.

**Pull request merged**